lib/rspec/core/configuration.rb

```diff
@@ -1350,20 +1350,31 @@ def exclusion_filter
       #       end
       #     end
       #
+      #     module PreferencesHelpers
+      #       def preferences(user)
+      #         # ...
+      #       end
+      #     end
+      #
       #     module UserHelpers
       #       def users(username)
       #         # ...
       #       end
       #     end
       #
       #     RSpec.configure do |config|
-      #       config.include(UserHelpers) # included in all groups
+      #       config.include(UserHelpers) # included in all examples
```

Comment:
To understand this, you have to realise that a group is a class and an example is an instance. So it is included in the group, which is then accessible in the instance.